platform/ios/platform/ios/test/CI.xctestplan

```diff
@@ -31,8 +31,7 @@
   "testTargets" : [
     {
       "skippedTests" : [
-        "MGLMapSnapshotterTests\/testLocalGlyphRendering",
-        "MGLOfflineStorageTests\/testAddPackForBounds"
+        "MGLMapSnapshotterTests\/testLocalGlyphRendering"
```

Comment:
Excellent.  Testing in the `CI` scheme works as well.  thanks for enabling the disabled tests.